pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
   flutter: ">=3.0.0"
 
 dependencies:
-  enough_platform_widgets: ^0.7.1
+  enough_platform_widgets: 0.6.0
```

Comment:
Fix for build error due to multiple import. Raised here https://github.com/Enough-Software/enough_platform_widgets/issues/12